src/app/app.routing.module.ts

```diff
@@ -0,0 +1,25 @@
+import { LoginComponent } from './auth/login/login.component';
```

Comment:
[Nice to have]: Imports organized, like:
1. Library - `@angular`, etc.
2. 3rd party - `@material`, etc.
3. Own code imports